app/views/site/_study_settings_viz.html.erb

```diff
@@ -0,0 +1,83 @@
+<% if @study.can_visualize_clusters? && @study.default_cluster.present? %>
+  <h3>Default view options</h3>
+  <p class="help-block">
+    Configure basic visualization defaults, such as what the default expression axis label, cluster, annotation,
+    color profile, and cluster point display properties are when someone views your study.
+  </p>
+  <div id="default-study-options-form">
+    <%= f.fields_for :default_options do |opts| %>
+      <div class="form-group row">
+        <div class="col-sm-4">
+          <%= opts.label :cluster, 'Default Cluster' %><br />
+          <%= opts.select :cluster, options_for_select(@study.cluster_groups.map(&:name), @default_cluster.name), {}, class: 'form-control' %>
+        </div>
+        <div class="col-sm-4">
+          <%= opts.label :annotation, 'Default Annotation' %><br />
+          <%= opts.select :annotation, grouped_options_for_select(@default_cluster_annotations, @study.default_annotation), {},class: 'form-control' %>
+        </div>
+        <div class="col-sm-4">
+          <%= opts.label :color_profile, "Default Color Profile <i class='fas fa-info-circle' title='The default color profile only applies to the selected annotation (if the type is numeric)' data-toggle='tooltip'></i>".html_safe %><br />
+          <%= opts.select :color_profile, options_for_select(SiteController::COLORSCALE_THEMES, @study.default_color_profile), {include_blank: 'N/A'} , class: 'form-control', disabled: @study.default_annotation_type == 'group' %>
+        </div>
+      </div>
+      <div class="form-group row">
+        <div class="col-sm-4">
+          <%= opts.label :cluster_point_size, "Size of Cluster Points (in px) <i class='fas fa-fw fa-question-circle' data-toggle='tooltip' title='Global value for the size of all cluster points, in pixels.'></i>".html_safe %><br />
+          <%= opts.number_field :cluster_point_size, value: @study.default_cluster_point_size, class: 'form-control' %>
+        </div>
+        <div class="col-sm-4">
+          <%= opts.label :cluster_point_border, 'Show Cluster Point Borders?' %><br />
+          <%= opts.select :cluster_point_border, options_for_select([['Yes', true],['No', false]], @study.show_cluster_point_borders?), {}, class: 'form-control' %>
+        </div>
+        <div class="col-sm-4">
+          <%= opts.label :cluster_point_alpha, "Cluster Point Opacity <i class='fas fa-fw fa-question-circle' data-toggle='tooltip' title='Global value for the transparency (alpha) of all cluster points.  Only values between 0 and 1 are accepted.'></i>".html_safe %><br />
```

Comment:
This is a good opportunity to update these overlooked labels to the new sentence case standard style.

```suggestion
          <%= opts.label :cluster, 'Default cluster' %><br />
          <%= opts.select :cluster, options_for_select(@study.cluster_groups.map(&:name), @default_cluster.name), {}, class: 'form-control' %>
        </div>
        <div class="col-sm-4">
          <%= opts.label :annotation, 'Default annotation' %><br />
          <%= opts.select :annotation, grouped_options_for_select(@default_cluster_annotations, @study.default_annotation), {},class: 'form-control' %>
        </div>
        <div class="col-sm-4">
          <%= opts.label :color_profile, "Default color profile <i class='fas fa-info-circle' title='The default color profile only applies to the selected annotation (if the type is numeric)' data-toggle='tooltip'></i>".html_safe %><br />
          <%= opts.select :color_profile, options_for_select(SiteController::COLORSCALE_THEMES, @study.default_color_profile), {include_blank: 'N/A'} , class: 'form-control', disabled: @study.default_annotation_type == 'group' %>
        </div>
      </div>
      <div class="form-group row">
        <div class="col-sm-4">
          <%= opts.label :cluster_point_size, "Size of cluster points (in px) <i class='fas fa-fw fa-question-circle' data-toggle='tooltip' title='Global value for the size of all cluster points, in pixels.'></i>".html_safe %><br />
          <%= opts.number_field :cluster_point_size, value: @study.default_cluster_point_size, class: 'form-control' %>
        </div>
        <div class="col-sm-4">
          <%= opts.label :cluster_point_border, 'Show cluster point borders?' %><br />
          <%= opts.select :cluster_point_border, options_for_select([['Yes', true],['No', false]], @study.show_cluster_point_borders?), {}, class: 'form-control' %>
        </div>
        <div class="col-sm-4">
          <%= opts.label :cluster_point_alpha, "Cluster point opacity <i class='fas fa-fw fa-question-circle' data-toggle='tooltip' title='Global value for the transparency (alpha) of all cluster points.  Only values between 0 and 1 are accepted.'></i>".html_safe %><br />
```